FGTranslator.podspec

```diff
@@ -8,7 +8,8 @@ Pod::Spec.new do |s|
   
   s.author       = { "George Polak" => "george.polak@gmail.com" }
 
-  s.platform     = :ios, "7.0"
+  s.ios.platform     = :ios, "7.0"
+  s.osx.platform     = :osx, "10.8"
```

Comment:
This part of a separate PR and not relevant to PINCache. Please remove it and I will merge.